1_crosswalk_fetch/src/munge_crosswalk.R

```diff
@@ -61,3 +61,44 @@ LAGOS_zip_to_sf <- function(out_ind, layer, zip_ind, ...){
   saveRDS(shp, data_file)
   gd_put(out_ind, data_file)
 }
+
+#' we take the MGLP geodatabase and read it into sf,
+#' then remove lakes that aren't part of the assessment and/or aren't
+#' part of the expansion footprint, as defined by state IDs in ...
+#'
+#' @param out_ind indicator file to write for output
+#' @param mglp_gdb_file geodatabase file for mglp footprint
+#' @param mglp_zip_ind indicator for zipped location of mglp footprint
+#' @param states states filter for mglp footprint
+#' @param ... other indicator files for those files (not explicitly inspected)
+#'   making up the shapefile
+munge_names <- function(out_ind, mglp_gdb_file, mglp_zip_ind, states, ...){
+
+  # get mglp lake info
+  zip_file <- scipiper::sc_retrieve(mglp_zip_ind)
+
+  shp.path <- tempdir()
+  unzip(zip_file, exdir = shp.path)
+
+  shp <- sf::st_read(file.path(shp.path, mglp_gdb_file), layer = 'MGLP_LAKES') %>%
+    filter(ASSESS == 'Y', STATE %in% states) %>%
+    mutate(site_id = paste0('mglp_', LAKE_ID))
+
+  mglp_lake_info <- data.frame(site_id = shp$site_id,
+                                  area_m2 = shp$SHAPE_Area,
+                                  lake_name = shp$LAKE_NAME)
+
+  # get winslow lake info
```

Comment:
Is Winslow data always supplied? if not, could add an if / else statement like this (I think this should work): 
```
if('winslow_shp_ind' %in% list(...)){
   [code you have]
    lake_info <- bind_rows(mglp_lake_info, winslow_lake_info)
}else{
    lake_info <- mglp_lake_info
}
```

Reply:
Yes, for now it is. This will likely be changed when we decide to start from scratch on the shapefiles and no longer need Winslow as a dependency.